docs/content/ingestion/firehose.md

```diff
@@ -36,6 +36,28 @@ A sample local firehose spec is shown below:
 |filter|A wildcard filter for files. See [here](http://commons.apache.org/proper/commons-io/apidocs/org/apache/commons/io/filefilter/WildcardFileFilter.html) for more information.|yes|
 |baseDir|directory to search recursively for files to be ingested. |yes|
 
+#### HttpFirehose
+
+This Firehose can be used to read the data from remote sites via HTTP.
+A sample http firehose spec is shown below:
+
+```json
+{
+    "type"    : "http",
+    "uris"  : ["http://example.com/uri1", "http://example2.com/uri2"]
+}
+```
+
+The below configurations can be optionally used for tuning the firehose performance.
+
+|property|description|default|
+|--------|-----------|-------|
+|maxCacheCapacityBytes|Maximum size of the cache space in bytes. 0 means disabling cache.|1073741824|
+|maxFetchCapacityBytes|Maximum size of the fetch space in bytes. 0 means disabling prefetch.|1073741824|
```

Comment:
Cache and fetch spaces are managed differently to achieve [different goals](https://github.com/druid-io/druid/pull/4193#discussion_r115721431). That is, cached files are not removed until the jvm terminates for future reads, while fetched files are removed when a connection on firehose is closed.

The goal of prefetching is to improve the throughput of the firehose, so ```maxFetchCapacityBytes``` should be tuned sometimes. For example, let me suppose an index task which reads data from an HttpFirehose and generates indexes. 

- If the indexing speed is so fast but http connection is so slow, reading from the firehose will be a bottleneck. In this case, increasing ```maxFetchCapacityBytes``` will be helpful.
- Or, if the indexing speed is not bad but the amount of remaining disk is very little, decreasing ```maxFetchCapacityBytes``` might be helpful.

Maybe better to add a document about this?